.github/workflows/code-ql-analysis.yml

```
@@ -0,0 +1,51 @@
+  push:
+    branches: [main]
+    paths:
+    - '**.cs'
+    - '**.csproj'
+  pull_request:
+    branches: [main]
+    paths:
+    - '**.cs'
+    - '**.csproj'
+  schedule:
+    - cron: '0 8 * * 4'
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)